src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -64,14 +65,26 @@ const WorkspacePermissionNotice = ({ workspace }) => {
   ])
 }
 
+const AzureWarning = () => {
+  const warningMessage = 'Reminder: Azure workspaces (like this one) are currently NOT authorized to contain any federally protected data. ' +
+    'Please do not import or use any federally protected data in Azure workspaces.'
+
+  return h(TitleBar, {
+    title: div({ style: { display: 'flex', alignItems: 'center', margin: '1rem' } }, [
+      icon('warning-standard', { size: 32, style: { color: colors.danger(), marginRight: '0.5rem' } }),
+      span({ style: { color: colors.dark() } }, [warningMessage])
+    ]), style: { backgroundColor: colors.accent(0.25) }
+  })
+}
+
 const WorkspaceTabs = ({
   namespace, name, workspace, isGoogleWorkspace, activeTab, refresh,
   setDeletingWorkspace, setCloningWorkspace, setSharingWorkspace, setShowLockWorkspaceModal, setLeavingWorkspace
 }) => {
   const isOwner = workspace && Utils.isOwner(workspace.accessLevel)
   const canShare = workspace?.canShare
   const isLocked = workspace?.workspace.isLocked
-  const isAzureWorkspace = !isGoogleWorkspace
+  const isAzureWorkspace = !!workspace?.workspace.azureContext
```

Comment:
`isAzureWorkspace = !isGoogleWorkspace` isn't accurate since `isGoogleWorkspace` is falsey when `workspace` has not loaded yet.

This is a more accurate `isAzureWorkspace` and removes the need for the use of `workspace &&` (in my prior commits).


this fixes an annoying visual effect: a brief flash of the banner while the workspace was being loaded (since isAzureWorkspace is defined as !isGoogleWorkspace), before disappearing.

Reply:
@andy7i you want `!!workspace?.azureContext` here.